### ISSUE TITLE
comments section doesn't minimized itself with player

### ISSUE THREAD

**Issue by NeeRaj-2401** [libre-tube/LibreTube#3631]
### Steps to reproduce

after #3624 

1. open comments section in portrait mode
2. swipe down player to mini player mode

### Expected behavior

comments section should close itself

### Actual behavior

mini players gets hidden behind the comments sheet as it is not closed

### LibreTube version

latest debug (PR #3624)

### Android version

android 12

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.
